x/stakeibc/types/params.go

```diff
@@ -27,6 +27,8 @@ var (
 	DefaultMaxStakeICACallsPerEpoch         uint64 = 100
 	DefaultIBCTransferTimeoutNanos          uint64 = 1800000000000 // 30 minutes
 	DefaultSafetyNumValidators              uint64 = 35
+	DefaultMinNumRebalance					uint64 = 1
```

Comment:
can you double check the linting in this file? the spacing seems off

Reply:
idk why but it's look fine on my side though 
![image](https://user-images.githubusercontent.com/63839106/220154887-8836500c-5a78-481c-ab2c-69fa9cff183a.png)